packages/storybook/stories/components/Textarea.stories.tsx

```diff
@@ -0,0 +1,173 @@
+/* @license CC0-1.0 */
+
+import type { Meta, StoryObj } from '@storybook/react';
+import { Textarea } from '@utrecht/component-library-react/src/css-module/Textarea';
+import readme from '@utrecht/components/textarea/README.md?raw';
+import tokensDefinition from '@utrecht/components/textarea/tokens.json';
+import tokens from '@utrecht/design-tokens/dist/index.json';
+import React from 'react';
+import { designTokenStory } from './util';
+
+const defaultArgs = {
+  dir: 'auto',
+  disabled: false,
+  focus: false,
+  hover: false,
+  invalid: false,
+  placeholder: undefined,
+  readOnly: false,
+  required: false,
+  spellCheck: false,
+  children: undefined,
+};
+
+const exampleArgs = {
+  children: 'The Quick Brown Fox Jumps Over The Lazy Dog',
+};
+
+const arabicDecorator = (Story) => (
+  <div dir="rtl" lang="ar">
+    {Story()}
+  </div>
+);
+
+const meta = {
+  title: 'CSS Component/Textarea',
+  id: 'css-textarea',
+  component: Textarea,
+  argTypes: {
+    disabled: {
+      description: 'Disabled',
+      control: 'boolean',
+    },
+    invalid: {
+      description: 'Invalid',
+      control: 'boolean',
+    },
+    readOnly: {
+      description: 'Read-only',
+      control: 'boolean',
+    },
+    required: {
+      description: 'Required',
+      control: 'boolean',
+    },
+  },
+  args: {},
+  tags: ['autodocs'],
+  parameters: {
+    status: {
+      type: 'WORK IN PROGRESS',
+    },
+    tokensPrefix: 'utrecht-textarea',
+    tokens,
+    tokensDefinition,
+    docs: {
+      description: {
+        story: `Styling via the \`.utrecht-textarea\` class name.`,
+        component: readme,
```

Comment:
By putting configuring `story` in the `meta` part, this same exact line of text becomes the documentation for each story. Each story should document the class name that makes it unique, not saying over and over again what is just the base class name. Best to move move this text to `Default.parameters`.